README.md

```diff
@@ -68,15 +68,15 @@ const App = () => {
 - [x] Search
 - [x] Slider
 - [x] Stepper
-- [ ] Switch
-- [ ] Uploader
+- [x] Switch
+- [x] Uploader
```

Comment:
Can you double-check if the uploader is up on the master branch? I don't see it in the storybook.